src/CMakeLists.txt

```diff
@@ -51,6 +51,20 @@ target_link_libraries(
 add_library(mp-units::mp-units ALIAS mp-units)
 install(TARGETS mp-units EXPORT mp-unitsTargets)
 
+# C++ module
+if(CMAKE_EXPERIMENTAL_CXX_MODULE_CMAKE_API)
```

Comment:
How is this being set? Shouldn't it be explicitly set somewhere to a proper UID?

Reply:
You can.
But the good practice is to pass it to the CMake configuration command.
That's the place where it belongs,
as it is the user who knows their version of CMake,
and them who should opt-into this experimental feature.

The Libc++ module for `std` doesn't do it this way, however: <https://libcxx.llvm.org/Modules.html#using-in-external-projects>.
When I use that, I need to comment it out because it doesn't match CMake vNext.

Reply:
Yes, I spoke with Bill Hoffman yesterday. He said the next release will remove the EXPERIMENTAL tag from C++ Modules.